bin/README.md

```diff
@@ -0,0 +1,10 @@
+# bin directory
+
+This directory is placed into `PATH` environment variable so that tools located
```

Comment:
I feel this can go in main docs since that serves as primary user guide for general usage.

Reply:
Where in the main docs would you mention it? Section in `writing_tests.md`?

Reply:
I think we can mention here https://github.com/red-hat-storage/ocs-ci/blob/master/docs/usage.md#deployment-of-cluster  that PATH is set after deployment of test and later versions of run will use *oc* binary from that PATH

Reply:
The rest of the docs can still stay here for someone trying to understand folder structure.

Reply:
As noted in the README, the PATH is set immediately in the run-ci wrapper, not after deployment, but you are right that it would be nice to mention it there.

Reply:
Addressed via: https://github.com/red-hat-storage/ocs-ci/pull/214/commits/a8c90bfe8ff63bf0f5066576a9ba99ad548a93fd